examples/gno.land/p/demo/blog/util.gno

```diff
@@ -5,3 +5,37 @@ import "strings"
 func breadcrumb(parts []string) string {
 	return "# " + strings.Join(parts, " / ") + "\n\n"
 }
+
+func merge(left, right []*Post) []*Post {
```

Comment:
Not a fan. Elaborating: I find that there is almost never a good reason to write out a function implementing a sorting algorithm (even if they are literally in all CS 101 courses), and `p/demo/blog` is likely to be one of the most visited realms at least for the upcoming year, so let's make sure its source code doesn't raise eyebrows and can instruct people on writing idiomatic Gno.

Use [`sort.Sort`](https://pkg.go.dev/sort#Sort), and create a `type postSlice []*Post` which implements [`sort.Interface`](https://pkg.go.dev/sort#Interface).